testsupport/space.go

```diff
@@ -188,7 +193,8 @@ func VerifyResourcesProvisionedForSpaceWithTiers(t *testing.T, awaitilities wait
 }
 
 func CreateMurWithAdminSpaceBindingForSpace(t *testing.T, awaitilities wait.Awaitilities, space *toolchainv1alpha1.Space, cleanup bool) (*toolchainv1alpha1.UserSignup, *toolchainv1alpha1.SpaceBinding) {
-	username := "for-space-" + space.Name
+	// username := "for-space-" + space.Name
+	username := space.Name // TODO: keep the "for-space-" prefix?
```

Comment:
yes please - keep the prefix. Since it's created automatically (behind the scene) then having the prefix is a good pointer for the reason why the MUR was created. I found it pretty useful when I was troubleshooting some e2e failures.

Reply:
restored in 7129ba8